### PR TITLE
Merge some changes from PR #40

### PR DESCRIPTION
**Pull request by phhai-hcmut**

### CODE REVIEW

File: devshell.nix [hunk 1]
```diff
@@ -6,10 +6,30 @@ let
...
+      flake8
+      flake8-isort
+      flake8-black
     '';
   };
 in
 {
   devshell.packages = [ pythonEnv ];
-  env = [{ name = "MYPY_CACHE_DIR"; eval = "$PRJ_DATA_DIR/mypy_cache"; }];
+  env = [
```
Comment (produdez) [inline devshell.nix:17]: Can you uh, explain the sorcery contents of this file?

File: flake.nix [hunk 2]
```diff
@@ -11,14 +11,54 @@
   };
 
   outputs = { self, nixpkgs, ... }@inputs:
-    nixpkgs.lib.flake.eachDefaultSystem (system:
-      let
-        pkgs = nixpkgs.legacyPackages.${system};
+    nixpkgs.lib.flake.eachSystem [ "x86_64-linux" ] (system:
```
Comment (produdez) [inline flake.nix:14]: And this file also :v

**Review by produdez — APPROVED** [review]

File: devshell.nix (hunk 1 above)
Comment (phhai-hcmut) [reply devshell.nix:17]: Just ignore them

Comment (produdez) [reply devshell.nix:17]: adou :v

**Pull request merged**